cluster-provision/k8s/1.18/kargs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -ex
+
+# OvS-DPDK requires latest kernel
+yum update -y --nobest kernel
+
+# Increase the Hugepages count so that DPDK can be enabled in OvS which requires 1GB of hugepages
+# Add iommu support to enable PMD drivers
+GRUB_FILE=/etc/default/grub
+MODIFIED=0
+if ! grep -q iommu $GRUB_FILE; then
+    sed -i 's/${GRUB_CMDLINE_LINUX}/${GRUB_CMDLINE_LINUX} iommu=pt intel_iommu=on/g' $GRUB_FILE
+    MODIFIED=1
+fi
+
+if grep -q 'hugepagesz=2M' $GRUB_FILE && ! grep -q 'hugepages=2048' $GRUB_FILE; then
```

Comment:
Typo in `hugepagesz`? Looks like 1.17 has it too...

Reply:
Can you be more specific? ``hugepagesz`` specifies the size whether 2M or 1G. ``hugepages`` specifies the number of hugepages of specifified size. The check here ensures that when 2M hugepage size is enabled, it should have atleast 2048 number of 2M hugepages, which allows 4GB of total hugepages for the deployment. 

1GB will be allocated to ovs-vswitch when DPDK is enabled. And the rest of the hugepages will be used in the VMs which uses vhostuser interfaces.

Reply:
My bad, thanks for the explanation.
/lgtm